### DISCUSSION TITLE
Document popup automatically display when entering insert mode

### DISCUSSION THREAD

**Discussion by bluzky** [helix-editor/helix#6710] [Q&A] [answered]
Hi everyone,

I'm new to Helix. I've just set up and try to edit an elixir source file. When I enter insert mode, the document popup for function under cursor automatically display. 
![Screenshot 2023-04-11 at 2 04 15 PM](https://user-images.githubusercontent.com/6194779/231082790-2e5ff96d-b236-4d62-b534-e7ada7c49bc1.png)

This is very annoying, cover up my source code. Is there anyway to disable or configure it?

Thanks

**Comment by h1t** [comment, marked as answer]
Add this snippet to your config:

[editor.lsp]
auto-signature-help = false

**Reply by bluzky** [reply]
Thanks @h1t, it works

**Comment by kemelzaidan** [comment]
I'd like to see that below the line, instead of above, so it would not hide the source code. Is there a way @h1t ?

**Reply by matta** [reply]
I don't think the placement is configurable.

You can type Ctrl-c to make popups go away, though.

**Reply by kemelzaidan** [reply]
> You can type Ctrl-c to make popups go away, though.

In insert mode? It's not working to me... Maybe I remapped the key... do you know the name for this action?

Thanks.

**Reply by matta** [reply]
I'm not very familiar with this aspect of Helix, but I do see Ctrl-c listed under the "Prompt" section of https://docs.helix-editor.com/keymap.html#insert-mode. There, the "Prompt" bindings are documented as not configurable.

I'm talking about the situation where Helix pops up docs as seen below. When I press Ctrl-c in the situation below, the doc popup goes away, and I remain in insert mode.

![Screenshot from 2024-07-05 14-34-31](https://github.com/helix-editor/helix/assets/37314/6e3f8520-76c2-481f-8a78-0048ba62b67d)
